catroid/src/androidTest/java/org/catrobat/catroid/test/utils/TestUtils.java

```diff
@@ -92,4 +97,76 @@ public static Pixmap createRectanglePixmap(int width, int height, Color color) {
 		pixmap.fillRectangle(0, 0, width, height);
 		return pixmap;
 	}
+
+	public static Project createProjectWithGlobalValues(String name, String spriteName, String valueName, Context context) {
+		Project project = new Project(context, name);
+		UserList firstUserList = new UserList(valueName);
+		UserVariable firstUserVariable = new UserVariable(valueName);
+
+		UserList secondUserList = new UserList(valueName + "_1");
+		UserVariable secondUserVariable = new UserVariable(valueName + "_1");
+
+		project.getDefaultScene().getProject().addUserList(firstUserList);
```

Comment:
this:
project.getDefaultScene().getProject().addUserList(firstUserList);
is the same as:
project.addUserList(firstUserList);

Reply:
Do you want it to be replaced by project.addUserList(firstUserList);?